### ISSUE TITLE
expandShorthand: true. I get an error, when shorthand property has a value unrelated to primary option.

### ISSUE THREAD

**Issue by nmatushevskiy** [AndyOGo/stylelint-declaration-strict-value#152]
Hello!

I created repo for reproduction. Please, check it:
https://github.com/nmatushevskiy/stylelint-declaration-strict-value-expandShorthand-issue

I want that in properties containing color the value to be as variable (like background-color, color, background, border etc).

To reach this, I use `stylelint-declaration-strict-value` plugin with `expandShorthand` property (see [stylelintrc.json](https://github.com/nmatushevskiy/stylelint-declaration-strict-value-expandShorthand-issue/blob/master/example.less) from repo for details).

I get an error, when in shorthand property has a value unrelated to color.

For example:

```
.example {
    border: 0; // unexpected error
}
```

see [example.less](https://github.com/nmatushevskiy/stylelint-declaration-strict-value-expandShorthand-issue/blob/master/example.less) from repo for details.

Is this expected behavior?

**Comment by AndyOGo** [comment]
Thank you for the issue @nmatushevskiy 

This is how [`shortcss`](https://www.npmjs.com/package/shortcss) expands your example:

```css
.example {
  border-width: 0; // valid expandsion
  border-style: 0; // invalid expandsion
  border-color: 0; // invalid expandsion
}
```

[`css-values`](https://www.npmjs.com/package/css-values) is used to validate if an expanded property is feasible.

This is a known limitation at the moment as stated in the docs:

https://github.com/AndyOGo/stylelint-declaration-strict-value#css-shorthand-syntax
> **Warning:** How do you determine, which longhand property is used by using a single shorthand? Like in this example:
> Differentiating this would need a CSS value parser and a curated list of valid CSS values per prop. This is beyond the scope of [`shortcss`](https://www.npmjs.com/package/shortcss).

Anyway if you think about all possible cases (which include variables).
The linter can't know the programmers intention for a shorthand variable.

**Comment by AndyOGo** [comment]
@nmatushevskiy 
I just published [`v1.7.11`](https://github.com/AndyOGo/stylelint-declaration-strict-value/releases/tag/v1.7.11).
Please let me know if that fixes your issue.

**Comment by nmatushevskiy** [comment]
Yes. Problem is fixed in the new version.
Thank you so much.
